gadgets/testing/testing.go

```diff
@@ -42,10 +60,25 @@ func RemoveMemlock(t testing.TB) {
 	require.NoError(t, err, "Failed to remove memlock rlimit: %s", err)
 }
 
+// MinimumKernelVersion skips the test if the current kernel version is less
+// than the minimum version. When using Kubernetes, it gets the kernel version
+// from a random node in the cluster.
```

Comment:
Was just about to write that this would give the kernel version of the test host. Thanks for implementing this :+1:

Reply:
It took me a while to understand why it wasn't working :D